src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -281,6 +281,98 @@ function Test-RemoveAzureStorageAccount
 
 <#
 .SYNOPSIS
+Test New-AzureRmStorageAccountEncryptionKeySource
+#>
+function Test-NewAzureRmStorageAccountEncryptionKeySource
+{
+    # Setup
+
+    try
+    {
+        # Test
+		$keySource = New-AzureRmStorageAccountEncryptionKeySource -Type MicrosoftStorage
+        Assert-AreEqual $keySource.KeySource MicrosoftStorage;
+        Assert-AreEqual $keySource.Keyvaultproperties.Keyname $null;
+        Assert-AreEqual $keySource.Keyvaultproperties.KeyVersion $null;
+        Assert-AreEqual $keySource.Keyvaultproperties.KeyVaultUri $null;
+		
+		$keyName = "testKeyName"
+		$keyVersion = "TestKeyversion"
+		$keyVaultUri = "https://test.vault.azure.net/"
+
+		$keySource = New-AzureRmStorageAccountEncryptionKeySource -Type MicrosoftKeyvault -KeyName $keyName -KeyVersion $keyVersion -KeyVaultUri $keyVaultUri
+        Assert-AreEqual $keySource.KeySource MicrosoftKeyvault;
+        Assert-AreEqual $keySource.Keyvaultproperties.Keyname $keyName;
+        Assert-AreEqual $keySource.Keyvaultproperties.KeyVersion $keyVersion;
+        Assert-AreEqual $keySource.Keyvaultproperties.KeyVaultUri $keyVaultUri;
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $rgname
+    }
+}
+
+<#
+.SYNOPSIS
+Test New-AzureRmStorageAccountEncryptionKeySource
+#>
+function Test-SetAzureRmStorageAccountKeySource
+{
+    # Setup
+    $rgname = Get-StorageManagementTestResourceName;
+
+    try
+    {
+        # Test
+        $stoname = 'sto' + $rgname;
+        $stotype = 'Standard_GRS';
+        $loc = 'westus';
```

Comment:
should not hard code the location, this prevents running a test agaionst different environments, or usign a subscription without access to this region